lib_nbgl/include/nbgl_use_case.h

```diff
@@ -137,10 +137,67 @@ typedef struct {
     uint8_t nbContents;  ///< number of contents
 } nbgl_genericContents_t;
 
+/**
+ * @brief The different types of operation to review
+ *
+ */
+typedef enum {
+    TYPE_OPERATION = 0,
+    TYPE_TRANSACTION,
+    TYPE_MESSAGE,
+} nbgl_operationType_t;
+
+/**
+ * @brief The different types of review status
+ *
+ */
+typedef enum {
+    STATUS_TYPE_OPERATION_SIGNED = 0,
+    STATUS_TYPE_OPERATION_REJECTED,
+    STATUS_TYPE_TRANSACTION_SIGNED,
+    STATUS_TYPE_TRANSACTION_REJECTED,
+    STATUS_TYPE_MESSAGE_SIGNED,
+    STATUS_TYPE_MESSAGE_REJECTED,
+    STATUS_TYPE_ADDRESS_VERIFIED,
+    STATUS_TYPE_ADDRESS_REJECTED,
+} nbgl_reviewStatusType_t;
```

Comment:
IS it needed to specify `STATUS_TYPE_XXXX`?
Couldn't we simply have something like:
```
typedef enum {
    STATUS_TYPE_SIGNED = 0,
    STATUS_TYPE_VERIFIED,
    STATUS_TYPE_REJECTED,
} nbgl_reviewStatusType_t;
```

Or even, it could be a simple `bool` ? because in fact `SIGNED` or `VERIFIED` means `SUCCESS`, or `VALIDATED`?

Reply:
It's needed to use the proper wording in the displayed screen